apps/docs/src/app/examples/checkbox/custom-icons/custom-icons.component.html

```diff
@@ -1,5 +1,5 @@
 <anglify-checkbox checked>Check me!</anglify-checkbox>
-<anglify-checkbox checked [ripple]="false">No ripple!</anglify-checkbox>
+<anglify-checkbox checked ripple="false">No ripple!</anglify-checkbox>
```

Comment:
The BooleanLike Type makes it possible to do just that. I think it's important that it's in the documentation so that the user sees that you can do this. This saves him some time and true attributes don't need a value at all anymore.